### ISSUE TITLE
Add support for postgresql-14 for Fedora

### ISSUE THREAD

**Issue by phracek** [sclorg/postgresql-container#442]
It would be nice to have support for postgresql-14 in our posgresql-image as well.
In order to support it, we need to modify these files:

- https://github.com/sclorg/postgresql-container/blob/master/specs/multispec.yml
- https://github.com/sclorg/postgresql-container/blob/master/src/Dockerfile.fedora
- https://github.com/sclorg/postgresql-container/blob/master/manifest.sh

See the similar pull request for support postgresql-container for CentOS Stream 8 here: https://github.com/sclorg/postgresql-container/pull/437

**Comment by phracek** [comment]
This pull request has to be merged before https://github.com/sclorg/s2i-base-container/pull/254

**Comment by Victoremepunto** [comment]
bump - since sclorg/s2i-base-container#254 has already been merge, would like to know if getting postgresql-14 support is blocked on anything else ? thanks !
